Ginger/GingerCoreNET/LiteDBFolder/LiteDbReportBase.cs

```diff
@@ -205,9 +225,24 @@ public class LiteDbActivity : LiteDbReportBase
     {
         public string ActivityGroupName { get; set; }
         public List<LiteDbAction> ActionsColl { get; set; }
+        //public List<Tuple<string, string>> ChildExecutableItemsCount { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)